### ISSUE TITLE
Relatively small rectosigmoid on fetal MRI

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11568]
**Preferred term label:**
Diminutive size of the rectosigmoid compared to the rest of the colon

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID 25677577

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hirschsprung's Disease

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay 
Adding new term: 
- ID: HP:6001474
- label: Abnormal rectosigmoid index
- parents: [Abnormal fetal gastrointestinal system morphology (HP:0034207)](https://hpo.jax.org/app/browse/term/HP:0034207)
- pmid: PMID:17164511|PMID:25677577|PMID:17502185
- def: The rectosigmoid (R/S) ratio, a measure comparing rectal to sigmoid colon diameter, is below the lower limit of normal on prenatal ultrasound. The rectosigmoid is normally the widest segment of the fetal colon. If the rectosigmoid is narrower than the ascending or transverse colon above it), this is called a reversed rectosigmoid ratio or abnormal rectosigmoid index. The RS ratio is calculated by dividing the transverse rectal diameter by the sigmoid diameter. An R/S ratio of 1 or more is considered normal.
- comment: An abnormal rectosigmoid ratio may indicate Hirschsprung disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11568
- synonyms: Reversed rectosigmoid ratio(NONE)
